app/web/src/App.tsx

```diff
@@ -19,6 +20,7 @@ function App() {
             <AuthProvider>
               <CssBaseline />
               <EnvironmentBanner />
+              <HtmlMeta noSuffix />
```

Comment:
Does this definitely get overwritten by later ones? (I mean, it does seem to work)

Reply:
This one will be used as default for pages without more specific override.

For example:
- user goes to Login page which will set title to 'Login'
- you go to some other page which does not have its own HtmlMeta
- if there is no 'root' react helmet component, the title would not re render and stays 'Login'